Meta.Tests.ps1

```diff
@@ -794,28 +814,47 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'Markdown' {
                                         "run 'npm install -g gulp' in order to have this " + `
                                         "text execute.")
             }
+
             if($optIn)
             {
                 $mdErrors | Should Be 0
             }
         }
 
-        # We're using this tool to delete the node_modules folder because it gets too long
-        # for PowerShell to remove.
-        $null = Start-Process `
-            -FilePath "npm" `
-            -ArgumentList @('install','rimraf','-g','--silent') `
-            -Wait `
-            -WorkingDirectory $PSScriptRoot `
-            -PassThru `
-            -NoNewWindow
-        $null = Start-Process `
-            -FilePath "rimraf" `
-            -ArgumentList @(Join-Path -Path $PSScriptRoot -ChildPath 'node_modules') `
-            -Wait `
-            -WorkingDirectory $PSScriptRoot `
-            -PassThru `
-            -NoNewWindow
+        <#
+            We're uninstalling the dependencies so that the node_modules folder
+            do not linger on a users computer if run locally.
+            Also, this fixes so that when there is a apostrophe in the path for
+            $PSScriptRoot, the node_modules folder is correctly removed.
+        #>
+        It 'Should not throw an error when uninstalling dependencies' {
+            Write-Verbose -Message "NPM is uninstalling Gulp. This may take a few moments." -Verbose
+
+            {
+                # Uninstalled npm packages in reverse order
+                $npmAdditionalAdditionalParametersForStartProcess['ArgumentList'] = 'uninstall gulp-concat -D'
```

Comment:
Can you separate the dependencies into their own `It` blocks rather than just the one big 'uninstalling dependencies' `It` block?